e2e_test/src/independent/client_tests.rs

```diff
@@ -362,3 +358,116 @@ fn client_start_test() {
         }
     }
 }
+
+//auto test about issue 1867
+#[test]
+#[serial]
+fn client_password_leak_test() {
+    let test_dir = "/tmp/e2e_test/client_tests/client_password_leak_test";
+    let _server = new_server(test_dir);
+
+    let args = vec!["-p", "123"];
+    let client_execute = ClientDef::new(args);
+    let mut client = client_execute.run();
+    let resp = client.write(&["show tables;\n"]);
+    if let Ok(resp) = resp {
+        assert_eq!(resp, "error: unrecognized subcommand \'123\'")
+    }
+
+    {
+        let args = vec![];
+        let client_execute = ClientDef::new(args);
+        let mut client = client_execute.run();
+        let resp = client.write(&["show tables;\n"]);
+        if let Ok(resp) = resp {
+            assert_eq!(&resp[0..10], "Query took")
+        }
+    }
+}
+
+//auto test about issue 784
+#[test]
+#[serial]
+fn client_join_generate_physical() {
+    let test_dir = "/tmp/e2e_test/client_tests/client_join_generate_physical";
+    let _server = new_server(test_dir);
+    let args = vec![];
+    let client_execute = ClientDef::new(args);
+    let mut client = client_execute.run();
+
+    let resp = client.write(&[&format!(
+        "\\w {}/e2e_test/test_data/oceanic_station.txt\n",
+        get_workspace_dir().as_os_str().to_str().unwrap()
+    )]);
+
+    if let Ok(resp) = resp {
+        assert_eq!(&resp[0..10], "Query took")
+    }
+
+    let resp = _server
+        .client
+        .post("http://127.0.0.1:8902/api/v1/sql?db=public", "WITH l as (SELECT date_trunc('day', time) AS day, avg (temperature) AS day_temperature, station
+        FROM sea
+        WHERE station = 'LianYunGang'
+        AND time >= '2022-01-15T00:00:00'
+        AND time
+        < '2022-02-16T00:00:00'
+        GROUP BY station, day),
+        x as
+        (
+        SELECT date_trunc('day', time) AS day, avg (speed) AS day_speed, station
+        FROM wind
+        WHERE station = 'XiaoMaiDao'
+        AND time >= '2022-01-15T00:00:00'
+        AND time
+        < '2022-02-16T00:00:00'
+        GROUP BY station, day)
+        SELECT * FROM l JOIN x ON x.day= l.day")
+        .unwrap();
+
+    assert_eq!(
+        resp.text().unwrap(),
+        "day,day_temperature,station,day,day_speed,station\n2022-02-06T00:00:00.000000000,66.94915254237289,LianYunGang,2022-02-06T00:00:00.000000000,65.55,XiaoMaiDao\n"
+    );
+}
+
+#[test]
+#[serial]
+#[ignore]
```

Comment:
why ignore

Reply:
this test can pass in my env.
MakeFile CI run this test: input_partitions=6, split_num=6
Local run this test: input_partitions=1, split_num=1
so i ignore it to pass the CI.
If this approach is not quite appropriate, I will make the necessary modifications.

Reply:
These configurations are also configurable locally

Reply:
the parameters that are not significantly relevant to the issue or bug have been handled.